corehq/apps/change_feed/data_sources.py

```diff
@@ -11,7 +12,7 @@
 FORM_SQL = 'form-sql'
 CASE_SQL = 'case-sql'
 SMS = 'sms'
-LEDGER = 'ledger'
+LEDGER_NEW = 'ledger-new'
```

Comment:
same thought applies everywhere in the PR, though not a big enough deal to block